### PR TITLE
37 add answers to two exercises

### PR DESCRIPTION
**Pull request by rogerkuou**
Hi @Peter9192, I added some answers for two exercises mentioned in #37. When you have time could you please check them? I am not sure I got everything correct but it is indeed fun to look into.

### CODE REVIEW

File: _episodes/access.md [hunk 1]
```diff
@@ -152,14 +152,100 @@ Next to large-scale general-use repositories such as [Zenodo](https://zenodo.org
...
+>>       <td>10. Is it easy to download data from this repository?</td>
+>>       <td>🥳Yes for data with open access.</td>
+>>       <td>🥳Yes.</td>
+>>       <td>🥳Yes.</td>
+>>     </tr>
+>>     <tr>
+>>       <td>11. Is it easy to upload data to this repository?</td>
+>>       <td>🥳Yes. Although one should use a Zenodo account, or an GitHub account, or an ORCID account.</td>
+>>       <td>🥳Yes. Although registering an account is required.</td>
+>>       <td>😢Not possible to upload.</td>
```
Comment (Peter9192) [inline _episodes/access.md]: I think these two have a complex procedure in place for if you want to upload data. You have to contact them, they will do checks, etc. That does lead to higher quality assurrence though.

**Review by Peter9192 — APPROVED** [review]
Nice work @rogerkuou ! Maybe you can add a little bit more info about how data gets uploaded to CDS or ESGF, but I'm also happy to see it merged as is.

File: _episodes/access.md (hunk 1 above)
Comment (rogerkuou) [reply _episodes/access.md]: Thx Peter! I edited the answer according to your advice. Could you please have another look?

**Review by Peter9192 — APPROVED** [review]

File: _episodes/access.md (hunk 1 above)
Comment (Peter9192) [reply _episodes/access.md]: Thanks!

**Pull request merged**